### PR TITLE
README: fixes, tweaks and re-organisation

### PR DESCRIPTION
**Pull request by hnez**
Hi,

I've spent some time with the README recently and found a few things that I would like to see changed.

Since changes in prose like a README are always a bit subjective I am of curse open for discussion. Feel free to point out changes you do not like.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -46,18 +46,18 @@ Packages also exist for `some other distributions <https://repology.org/project/
 Quickstart
 ----------
 
-Create and activate a virtualenv for usbsdmux:
+Create and activate a Python virtual environment for usbsdmux:
```
[the remark this answers is not in the document]
Comment (hnez) [reply README.rst]: I should give that a try, I've not used it yet.
Should we add it as an additional option or as a replacement for the manual venv route?

[the remark this answers is not in the document]
Comment (jluebbe) [reply README.rst]: I'd suggest using it as the primary option and keep manual venv for now.

**Comment by SmithChart** [comment]
These changes look good to me. Thanks for taking the time to improve the README!

Once the last thread is resolved this should be ready to merge.

**Review by SmithChart — APPROVED** [review]
We will merge this for now. If we want to update the `pipx` section later on we are free to add another commit.

**Pull request merged**